hack/generator/pkg/astmodel/package_import_set.go

```diff
@@ -269,7 +268,8 @@ func (set *PackageImportSet) ServiceNameForImport(imp PackageImport) string {
 //      for microsoft.storage/v20200101 extract "storagev20200101" and so on
 func (set *PackageImportSet) versionedNameForImport(imp PackageImport) string {
 	service := set.ServiceNameForImport(imp)
-	return service + imp.packageReference.PackageName()
+	version := imp.packageReference.PackageName()
+	return service + strings.Title(version)
```

Comment:
Is it standard practice for package names in imports to be title-cased? According to https://blog.golang.org/package-names package names should be all lowercase all the time - I had assumed that extended to when you're naming a package something other than the default name during import too?